playbooks/approve_add_host.yml

```diff
@@ -0,0 +1,6 @@
+- hosts: bastion
+  roles:
+    - role: approve_add_host
+      when: groups['day2_workers'] | default([]) | length > 0
+  vars:
+    - disconnected: "{{ use_local_mirror_registry | default(setup_registry_service | default(true)) }}"

```

Comment:
Where is this used?

Reply:
This is used for approving and adding host to cluster after node is booted with iso and installed

Reply:
deleted